index.html

```diff
@@ -30,8 +30,7 @@
     </header>
```

Comment:
Please write a detailed accessibility report here in the Pull request summary.

The report should include a check about page titles.
The report should include a check about image text alternatives (<img alt="  ">).
The report should include a check about text headings.
The report should include a check about color contrast.
The report should include a check about resizing.
The report should include a check about interaction.
The report should include a check about moving content.
The report should incldue a check about multimedia.
The report should include a check about the basic structure of the page.
In the result section of the report the student should include a statement pertained to every point checked:
  If everything is ok, please make a statement about that (e.g. "Page titles - no issues found").
  If issues are found, please add a link to the corresponding line of code in the Pull Request changes.